Userland/Demos/Mandelbrot/Mandelbrot.cpp

```diff
@@ -166,14 +190,26 @@ class MandelbrotSet {
         return *m_bitmap;
     }
 
+    double pixel_to_real(int pixel_x) const
+    {
+        return pixel_x * (m_x_end - m_x_start) / m_bitmap->width() + m_x_start;
+    }
+
+    double pixel_to_imag(int pixel_y) const
+    {
+        return pixel_y * (m_y_end - m_y_start) / m_bitmap->height() + m_y_start;
+    }
+
+    SetType m_set_type;
+
```

Comment:
All of that should be `private`. (That can be done once you put `g_c*` as a member of the class)